### PR TITLE
Feat/civ 849 damages

### PR DESCRIPTION
**Pull request by drummondjm**
**Before creating a pull request make sure that:**

- [*] commit messages are meaningful and follow good commit message guidelines
- [*] README and other documentation has been updated / added (if needed)
- [*] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/DefaultJudgementHandler.java [hunk 1]
```diff
@@ -80,6 +83,10 @@ private CallbackResponse validateDateValues(CallbackParams callbackParams) {
         List<Element<HearingDates>> hearingDatesElement = caseData.getHearingSupportRequirementsDJ().getHearingDates();
         List<String> errors = (Objects.isNull(hearingDatesElement)) ? null :
             isValidRange(hearingDatesElement);
+
```
Comment (sankhajuria) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/DefaultJudgementHandler.java:86]: Please remove these comments

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/DefaultJudgementHandlerTest.java [hunk 2]
```diff
@@ -135,6 +135,47 @@ void shouldReturnOne_whenHaveOneDefendants() {
         }
     }
 
+    @Nested
+    class MidEventHearingTypeSelection {
+        private static final String PAGE_ID = "hearingTypeSelection";
+
+        @Test
+        void shouldReturnDisposalText_whenHearingTypeSelectionDisposal(){
+            String DISPOSAL_TEXT = "will be disposal hearing provided text";
```
Comment (sankhajuria) [inline src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/DefaultJudgementHandlerTest.java:144]: Please add comment here for clarity .

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/DefaultJudgementHandler.java (hunk 1 above)
Comment (drummondjm) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/DefaultJudgementHandler.java:86]: Removed

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/DefaultJudgementHandlerTest.java (hunk 2 above)
Comment (drummondjm) [reply src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/DefaultJudgementHandlerTest.java:144]: Added clarifying comments

**Pull request merged**